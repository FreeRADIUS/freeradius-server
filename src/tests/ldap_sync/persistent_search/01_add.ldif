#
# ARGV: -x -H ${PERSISTENT_SEARCH_TEST_SERVER} -D "cn=manager" -w "secret123"
#
dn: uid=fred,ou=people,dc=example,dc=com
changeType: add
objectClass: inetOrgPerson
objectClass: posixAccount
objectClass: shadowAccount
uid: fred
sn: Smith
givenName: Fred
cn: Fred Smith
displayName: Fred Smith
userPassword: testing
uidNumber: 102
gidNumber: 102
homeDirectory: /home/fred

