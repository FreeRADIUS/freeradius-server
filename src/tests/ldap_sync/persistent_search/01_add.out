Add ou=people,dc=example,dc=com uid=fred,ou=people,dc=example,dc=com  fred
